src/xcode/ENA/ENA/Source/Services/Risk/Calculation/__tests__/RiskCalculationTests.swift

```diff
@@ -21,7 +21,8 @@
 import ExposureNotification
 import XCTest
 
-final class RiskCalculationTests: XCTestCase {
+// swiftlint:disable:next type_body_length
+class RiskCalculationTests: XCTestCase {
```

Comment:
Is `final` an option here?

```suggestion
final class RiskCalculationTests: XCTestCase {
```